routes/index.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports = (app, db) ->
+  app.get "/", (req, res) ->
+    res.send 'index'
```

Comment:
Can you make sure there are newlines at the end of files? vim does this for me automatically, but the red no sign on the end of line 3 tells me that you editor didn't. There are a handful of tools in linux land that want newlines as the last characters in files so you get things like this guy

![screen shot 2015-02-20 at 9 56 20 am](https://cloud.githubusercontent.com/assets/620189/6288401/be1c5964-b8e6-11e4-979c-41ed34576f82.png)

when you don't have them.